### PR TITLE
[OpenVINO EP] Minor bug fixes

### PR DESCRIPTION
**Pull request by suryasidd**
**Description**: Minor bug fixes for accelerators. Added dimensionality checks and disabled some unit tests for MYRIAD and GPU

**Motivation and Context**

- This change is required for running some of the models on OpenVINO EP instead of falling back to default CPU EP

### CODE REVIEW

File: onnxruntime/core/providers/openvino/openvino_execution_provider.cc [hunk 1]
```diff
@@ -89,7 +89,7 @@ bool IsDimensionSupported(const Node* node, std::string dev_id){
     if(node->OpType().find("Pool") != std::string::npos){
 
         if(dev_id == "MYRIAD" || dev_id == "HDDL"){
-            if(input_dims != 3 || input_dims != 4)
```
Comment (yufenglee) [inline onnxruntime/core/providers/openvino/openvino_execution_provider.cc]: change || to &&?

File: onnxruntime/core/providers/openvino/openvino_execution_provider.cc [hunk 2]
```diff
@@ -216,17 +216,26 @@ bool IsGraphSupported(const onnxruntime::GraphViewer& graph_viewer, std::string
...
 
-  if (num_inputs != 0)
-    input_dims = graph_proto->input(0).type().tensor_type().shape().dim_size();
+    //GPU Plugin does not support 1D and 5D input
+    for(int i = 0; i < num_inputs; i++){
+        input_dims = graph_proto->input(i).type().tensor_type().shape().dim_size();
 
-  if (num_outputs != 0)
-    output_dims = graph_proto->output(0).type().tensor_type().shape().dim_size();
+        if(dev_id == "GPU"){
```
Comment (yufenglee) [inline onnxruntime/core/providers/openvino/openvino_execution_provider.cc]: move if(dev_id == "GPU") outside of for loop

File: onnxruntime/core/providers/openvino/openvino_execution_provider.cc [hunk 3]
```diff
@@ -216,17 +216,26 @@ bool IsGraphSupported(const onnxruntime::GraphViewer& graph_viewer, std::string
...
+            if(input_dims == 1 || input_dims == 5)
+                return false;
+        }
+    }
+
+    //GPU Plugin does not support 5D output
+    for(int i = 0; i < num_outputs; i++){
+        output_dims = graph_proto->output(i).type().tensor_type().shape().dim_size();
+
+        if(dev_id == "GPU"){
```
Comment (yufenglee) [inline onnxruntime/core/providers/openvino/openvino_execution_provider.cc]: ditto

**Comment by jywu-msft** [comment]
looks like branch is out of date

**Pull request merged**